packages/block-library/src/comments-pagination-previous/block.json

```diff
@@ -0,0 +1,35 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "core/comments-pagination-previous",
+	"title": "Previous Page",
+	"category": "theme",
+	"parent": [ "core/comments-pagination" ],
+	"description": "Displays the previous comments page link.",
+	"textdomain": "default",
+	"attributes": {
+		"label": {
+			"type": "string"
+		}
+	},
+	"usesContext": [ "postId", "paginationArrow" ],
```

Comment:
Shouldn't `paginationArrow` be scoped to comments? It's probably less important here because there is no risk it would clash with Query Pagination block because they will always be in a different tree, but maybe that should be a good practice we follow. I'm fine either way, but I wanted to raise it as something where the block context needs more thought as we expand its usage.

Reply:
Totally agree, let's do that.

Reply:
We have a similar issue with `get_query_pagination_arrow()`, in this case is using `$block->context['paginationArrow']` for both Query and Comments Loop.

Let's create another function so we can solve both issues at the same time.